cyral/internal/repository/accessrules/schema_loader.go

```diff
@@ -0,0 +1,26 @@
+package accessrules
+
+import (
+	"github.com/cyralinc/terraform-provider-cyral/cyral/core"
+)
+
+type packageSchema struct {
+}
+
+func (p *packageSchema) Name() string {
+	return "repository_access_rules"
```

Comment:
```suggestion
	return "accessrules"
```

Reply:
Thanks! Addressed in 1d4545f311c3daf857fc7a7ab532fac26aa70c62.